locales/nl.json

```diff
@@ -12,7 +12,7 @@
 	"would turn %s off": "schakelt %s uit",
 	"would turn %s on": "shakelt %s in",
 	"if": "als",
-	"Edit rule": "Regels bewerken",
+	"Edit rule": "Regel bewerken",
```

Comment:
This isn't wrong.
The button in the menu gives access to multiple rules not just one.
if one rule then "Regel bewerken" is correct but multiple rules should be "Regels bewerken".